requirements.txt

```diff
@@ -6,3 +6,4 @@ matplotlib
 numpy
 requests
 shapely
+netrc
```

Comment:
This isn't necessary is it? 'netrc' is in the standard library since Python 1.5

Reply:
@wallinb you're right i removed the netrc requirement.  i also rebased into a single commit and am using urs.earthdata.nasa.gov following the NSIDC programmatic guide